Makefile

```diff
@@ -26,6 +26,8 @@ build_image_%:
 build_images: $(addprefix build_image_, $(IMAGES))
 push_images: $(addprefix push_image_, $(IMAGES))
 
+build_image_log_handler: docker/log_handler/requirements.txt
```

Comment:
i don't see a `docker/log_handler/requirements.txt` in this PR... forget to commit?

Reply:
or a `requirements.in`...

Reply:
sorry. I made a dev image. I'll remove this. Planning to squash so the history won't be to messy.